workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/metrics.py

```diff
@@ -46,7 +50,23 @@ def prepend_to_key(d, prefix):
 
 
 def weighted_mean(ds, w, dims):
-    return (ds * w).sum(dims) / w.sum(dims)
+    return (ds * w).sum(dims, skipna=True) / w.sum(dims, skipna=True)
```

Comment:
do we need this change? the masking sets the area (ie weights) to 0 outside the region of interest, so don't think we need to worry about nans.